### ISSUE TITLE
ScriptType

### ISSUE THREAD

**Issue by athos-ribeiro** [c9s/Vimana#19]
Whenever a plugin has no script type defined, Vimana does not know what to do with it.

Since many of the recent plugins just try to support pathogen whenever they are developed, they do not care about setting a type anymore. How should we address that? Should we just add the type to every project we want to support, or should we check for the project directory tree, and if they are pathogen compatible, distribute them according to the directory tree?

e.g.: If foobar project has a foobar/syntax/syntax.vim file, we could assume this is a syntax plugin.

**Comment by c9s** [comment]
Sure, I think it works. It would be great if you have any PR for this, I am glad to accept the patch. :)

**Comment by athos-ribeiro** [comment]
Definitely! I will write one this week, I created the issue 1st just to
make sure it was a good idea :)
Em 27/01/2015 11:39, "Yo-An Lin" <notifications@github.com> escreveu:
